.github/ISSUE_TEMPLATE.md

```diff
@@ -40,7 +40,7 @@
 
 <!--
     You have to add it, we can't help you otherwise.
-    If you can't pass it, send it via mail at mathieu.aa@free.fr
```

Comment:
Can you use the jhipster tech address instead? that'd be better :)

Reply:
mathieu.aa@jhipster.tech ?

Reply:
I believe it is `mathieu.abou-aichi@jhipster.tech`